app/code/Magento/Security/Model/AdminSessionsManager.php

```diff
@@ -34,7 +39,7 @@ class AdminSessionsManager
     protected $securityConfig;
 
     /**
-     * @var \Magento\Backend\Model\Auth\Session
+     * @var Session
```

Comment:
Can you please add a full class reference here as we are following the same in Core and it's easy to identify class.